src/styles/BooksList.css

```diff
@@ -0,0 +1,5 @@
+.books-container {
+  margin: 0;
+  padding: 0;
+  list-style: none;
+}
```

Comment:
- [ ] At the moment, there is a horizontal scroll on your page which is likely caused by the extended width of the containers. Please kindly remove the horizontal scroll as it does not give a good user experience.

![image](https://github.com/Sparktan24/Bookstore/assets/43995830/e3bd2f78-cb95-4563-b0b8-9fbd57777b5f)